src/plugins/workspace/public/components/workspace_form/workspace_feature_selector.tsx

```diff
@@ -19,21 +19,20 @@ import { PublicAppInfo } from '../../../../../core/public';
 import { isWorkspaceFeatureGroup, convertApplicationsToFeaturesOrGroups } from './utils';
 
 export interface WorkspaceFeatureSelectorProps {
-  applications: Array<
-    Pick<PublicAppInfo, 'id' | 'title' | 'category' | 'chromeless' | 'navLinkStatus'>
-  >;
   selectedFeatures: string[];
   onChange: (newFeatures: string[]) => void;
+  workspaceConfigurableApps?: PublicAppInfo[];
```

Comment:
Seems we only need id, title and category for workspaceConfigurableApps here. Could we change to `Pick<PublicAppInfo, 'id' | 'title' | 'category'>`?

Reply:
Actually I don't see a strong need to turn it into `Pick<PublicAppInfo, 'id' | 'title' | 'category'>` in this case. I understood that passing minimum set of data to child components is a best practice, but the `application ` which type is `PublicAppInfo` is provided by the core module, the components expect `PublicAppInfo` type of data sounds reasonable to me as well.

I'm not holding a strong opinion, what else concerns do you have?

Reply:
Not a big issue. We can keep current implementation.